### PR TITLE
Add vagrant v1.6.5

### PR DESCRIPTION
**Pull request by dtjm**

### CODE REVIEW

**Comment by dtjm** [comment]
Looks like I got this. Error. Any tips on debugging this?

```
$ /System/Library/Frameworks/Ruby.framework/Versions/"${CASK_RUBY_TEST_VERSION}"/usr/bin/bundle exec "/System/Library/Frameworks/Ruby.framework/Versions/${CASK_RUBY_TEST_VERSION}/usr/bin/rake" test
/Users/travis/build/caskroom/homebrew-versions/test/test_helper.rb:31:in `<top (required)>': uninitialized constant HOMEBREW_LIBRARY (NameError)
	from /Users/travis/build/caskroom/homebrew-versions/test/Casks/compliance_test.rb:1:in `require'
	from /Users/travis/build/caskroom/homebrew-versions/test/Casks/compliance_test.rb:1:in `<top (required)>'
	from /Library/Ruby/Gems/2.0.0/gems/rake-10.0.4/lib/rake/rake_test_loader.rb:10:in `require'
	from /Library/Ruby/Gems/2.0.0/gems/rake-10.0.4/lib/rake/rake_test_loader.rb:10:in `block (2 levels) in <main>'
	from /Library/Ruby/Gems/2.0.0/gems/rake-10.0.4/lib/rake/rake_test_loader.rb:9:in `each'
	from /Library/Ruby/Gems/2.0.0/gems/rake-10.0.4/lib/rake/rake_test_loader.rb:9:in `block in <main>'
	from /Library/Ruby/Gems/2.0.0/gems/rake-10.0.4/lib/rake/rake_test_loader.rb:4:in `select'
	from /Library/Ruby/Gems/2.0.0/gems/rake-10.0.4/lib/rake/rake_test_loader.rb:4:in `<main>'
rake aborted!
```

File: Casks/vagrant165.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'vagrant165' do
+  version '1.6.5'
+  sha256 'a94a16b9ed5f63460f64110738067aea029238f8d826c8dd90c5c34615a5be1e'
+
+  url "https://dl.bintray.com/mitchellh/vagrant/vagrant_#{version}.dmg"
+  homepage 'http://www.vagrantup.com'
+  license :unknown
```
Comment (vitorgalvao) [inline Casks/vagrant165.rb]: `:mit`.

https://github.com/caskroom/homebrew-cask/blob/da3c3a5be2b2b7ccbeadde04ab00277cfeb83888/Casks/vagrant.rb#L7

File: Casks/vagrant165.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+  pkg 'Vagrant.pkg'
+  uninstall :script => { :executable => 'uninstall.tool', :input => %w[Yes] }
```
Comment (vitorgalvao) [inline Casks/vagrant165.rb]: Missing a [line](https://github.com/caskroom/homebrew-cask/blob/da3c3a5be2b2b7ccbeadde04ab00277cfeb83888/Casks/vagrant.rb#L12).

**Comment by vitorgalvao** [comment]
The error might be a Travis hiccup. I’ve restarted the check.

**Comment by dtjm** [comment]
@vitorgalvao thanks for the feedback. I've made the changes you requested.

File: Casks/vagrant165.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  sha256 'a94a16b9ed5f63460f64110738067aea029238f8d826c8dd90c5c34615a5be1e'
+
+  url "https://dl.bintray.com/mitchellh/vagrant/vagrant_#{version}.dmg"
+  homepage 'http://www.vagrantup.com'
+  license :mit
+
+  pkg 'Vagrant.pkg'
+  uninstall :script => { :executable => 'uninstall.tool', :input => %w[Yes] },
+            :pkgutil => 'com.vagrant.vagrant'
+
```
Comment (vitorgalvao) [inline Casks/vagrant165.rb]: Please remove this empty line.

**Comment by vitorgalvao** [comment]
Please [squash your commits](http://davidwalsh.name/squash-commits-git).

**Comment by dtjm** [comment]
Removed line and force-pushed a new squashed commit.

**Comment by vitorgalvao** [comment]
Thank you.

**Pull request merged**